debian/debathena-pyhesiodfs.service

```diff
@@ -0,0 +1,20 @@
+# Author: Lizhou Sha <slz@mit.edu>
+# This file is part of the debathena-pyhesiodfs package.
+
+[Unit]
+Description=Hesiod automounter for Athena lockers
+Requires=openafs-client.service
+After=local-fs.target network.target openafs-client.service
+Before=umount.target
+
+[Service]
+Type=simple
+Environment="pyhesiodfs_dir=/mit"
+OOMScoreAdjust=-1000
+ExecStartPre=/bin/mkdir -p ${pyhesiodfs_dir}
+ExecStartPre=/bin/chown root:pyhesiodfs ${pyhesiodfs_dir}
+ExecStartPre=/bin/chmod 770 ${pyhesiodfs_dir}
+ExecStart=/usr/bin/pyhesiodfs -f ${pyhesiodfs_dir} -o nonempty
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Actually, since I have `Requires=openafs-client.service`, `pyhesiodfs` will be killed when `openafs-client` stops for any reason. Can we do `WantedBy=openafs-client.service` instead?